INSTALL.md

```diff
@@ -1,11 +1,14 @@
 Installing From Sources
 =======================
 
-This document presents instructions to install this branch of Coq.
-For more general installation instructions and information about known
-build system issues, please consult the wiki page: 
+The recommended way to install and use Coq is to install the Coq
+platform or to rely on a package manager (e.g. opam or Nix).
 
-  https://github.com/coq/coq/wiki#coq-installation
+See https://coq.inria.fr/download and
+https://github.com/coq/coq/wiki#coq-installation to learn more.
+
+Otherwise, the remainder of this file gives technical instructions for
+building Coq from sources.
```

Comment:
```suggestion
Otherwise, the remainder of this file explains how to build Coq from sources.
```

Reply:
The goal of this introduction was to discourage normal users from reading this file / attempting to install from sources. It was already alleviated by the suggestion of Hugo that I've applied (leading to the current sentence). If you remove "technical" from it, then it loses the word that was supposed to be discouraging. I'm open to other proposals.

Reply:
Maybe a bit much to expect from the single word "technical".
How about this?  Too unfriendly?  OTOH, I don't want to discourage anyone who is willing to spend the time to learn about Coq internals.

```suggestion
Otherwise, the remainder of this file explains how to build Coq from sources.  We recommend using this approach only if you plan to make changes to Coq and are willing to work through any issues with limited assistance.  
```

Reply:
That's too much indeed. We are willing to provide as much assistance as people need if they want to contribute. I'll push some changes. Let me know what you think.